packages/core/useScriptTag/index.ts

```diff
@@ -46,6 +46,8 @@ export interface UseScriptTagOptions extends ConfigurableDocument {
   attrs?: Record<string, string>
 }
 
+const store = new Map<string, number>()
```

Comment:
I am not sure about this internal global map. This will break tree-shaking and make the fix not applicable if the user has multiple versions of VueUse installed.

Reply:
To be honest I didn't think about it. Do you have any ideas about it ?

Reply:
> I am not sure about this internal global map. This will break tree-shaking and make the fix not applicable if the user has multiple versions of VueUse installed.

To be honest I didn't think about it. Do you have any ideas about it ?